test/pnp.test.js

```diff
@@ -112,15 +138,19 @@ describe("pnp", () => {
 		pnpApi.mocks.set("pkg", path.resolve(fixture, "pkg"));
 		resolver.resolve({}, __dirname, "pkg", {}, (err, result) => {
 			if (err) return done(err);
-			expect(result).toEqual(path.resolve(fixture, "pkg/main.js"));
+			expect(result).toEqual(
+				transferPathToPosix(path.resolve(fixture, `pkg${obps}main.js`))
+			);
 			done();
 		});
 	});
 	it("should resolve namespaced module names", done => {
-		pnpApi.mocks.set("@user/pkg", path.resolve(fixture, "pkg"));
-		resolver.resolve({}, __dirname, "@user/pkg", {}, (err, result) => {
+		pnpApi.mocks.set(`@user${posixSep}pkg`, path.resolve(fixture, "pkg"));
```

Comment:
thoughts: i'm not sure about using `posixSep` in this place, because it should use `osBasedPathSeparator (obps)` with the reasoning from `/test/util/path-separator`